src/main/java/net/minecraftforge/server/command/CommandTrack.java

```diff
@@ -19,19 +19,38 @@
 package net.minecraftforge.server.command;
 
 import javax.annotation.Nullable;
-import java.util.Collections;
-import java.util.List;
+import java.text.DecimalFormat;
+import java.util.*;
```

Comment:
Please avoid using star imports.

Reply:
they are used in a few other places across forge files.